src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -163,37 +193,40 @@ void merge_config_with(ov::AnyMap& lhs, const ov::AnyMap& rhs) {
     }
 }
 
-ov::AnyMap get_default_prefill_config() {
-    std::map<std::string, std::string> config = {
+ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model) {
+    ov::AnyMap config = {
         { "NPU_USE_NPUW", "YES" },
         { "NPUW_FOLD", "YES" },
         { "NPUW_DCOFF_TYPE", "f16" },
         { "NPUW_DCOFF_SCALE",  "YES" },
+        { "NPUW_WEIGHTS_BANK",  "shared" },
         { "NPUW_ONLINE_AVOID", "P:RMSNorm/NPU" }
```

Comment:
`AVOID`s are NOT required when DQ is in place. So please remove `AVOID`s from the default config if you enable DQ.

Reply:
Done